builder/builder/src/main/java/io/helidon/builder/RequiredAttributeVisitor.java

```diff
@@ -71,6 +90,7 @@ public void visit(String attrName,
      *
      * @throws java.lang.IllegalStateException when any attributes are in violation with the validation policy
      */
+    // important note: this needs to remain public since it will be new'ed from code-generated builder processing ...
```

Comment:
The comment is probably wrong here? `new'ed` does not make sense in this context...

Reply:
done in https://github.com/helidon-io/helidon/pull/5591/files